### PR TITLE
feat: add new git configs

### PR DESCRIPTION
**Pull request by katcipis**
Add new configs to git block:

```
check_untracked   = true
check_uncommitted = true
check_remote      = true
```

When the attributes are not defined in the git block they default to true. Also improved some attributes that had the wrong range when eval failed (wrong type).

### CODE REVIEW

File: hcl/hcl.go [hunk 1]
```diff
@@ -1117,8 +1135,6 @@ func parseGitConfig(git *GitConfig, gitBlock *ast.MergedBlock) error {
 
 			git.DefaultBranch = value.AsString()
 		case "default_remote":
-			logger.Trace().Msg("Attribute name was 'default_remote'.")
-
 			if value.Type() != cty.String {
 				errs.Append(errors.E(attr.NameRange,
```
Comment (i4ki) [inline hcl/hcl.go:1139]:
```suggestion
				errs.Append(errors.E(attr.Expr.Range(),
```
not related to your change but just noticed this is wrong here.

Comment (katcipis) [reply hcl/hcl.go:1139]: Extracted on a function trying to help being consistent about this.

File: hcl/hcl.go [hunk 2]
```diff
@@ -1456,3 +1501,11 @@ func listTerramateDirs(dir string) ([]string, error) {
 func isTerramateFile(filename string) bool {
 	return strings.HasSuffix(filename, ".tm") || strings.HasSuffix(filename, ".tm.hcl")
 }
+
+func hclAttrEvalErr(attr *hclsyntax.Attribute, msg string, args ...interface{}) error {
```
Comment (i4ki) [inline hcl/hcl.go:1505]: top

**Review by i4ki — APPROVED** [review]
LGTM

**Pull request merged**